rs/nns/gtc/BUILD.bazel

```diff
@@ -144,11 +138,14 @@ rust_test(
 
 rust_test(
     name = "canister_unit_test",
-    crate = ":_wasm_genesis-token-canister",
-    data = ["canister/gtc.did"],
-    env = {
-        "CARGO_MANIFEST_DIR": "rs/nns/gtc",
-    },
+    srcs = glob(["canister/**/*.rs"]),
+    compile_data = ["canister/gtc.did"],
+    crate_root = "canister/canister.rs",
+    proc_macro_deps = MACRO_DEPENDENCIES,
+    deps = DEPENDENCIES + DEV_DEPENDENCIES + [
+        ":build_script",
+        ":gtc",
+    ],
```

Comment:
crate is replaced with srcs + crate_root + deps. This is necessary because we broke out tests into their own file.

data + env is replaced with compile_data

proc_marco_deps is added just for consistency, and seems to make no difference one way or the other.